editor/components/rich-text/tinymce.js

```diff
@@ -163,14 +163,15 @@ export default class TinyMCE extends Component {
 			browser_spellcheck: true,
 			entity_encoding: 'raw',
 			convert_urls: false,
-			inline_boundaries_selector: 'a[href],code,b,i,strong,em,del,ins,sup,sub',
+			inline_boundaries_selector: 'a[href],code,b,i,strong,em,del,ins,sup,sub,.autocomplete-token',
 			plugins: [],
 			formats: {
 				strikethrough: { inline: 'del' },
 			},
 		} );
 
-		settings.plugins.push( 'paste' );
+		settings.plugins.push( 'paste', 'noneditable' );
+		settings.noneditable_noneditable_class = 'readonly-token';
```

Comment:
So we're making this non editable again?

Reply:
The latest change treats completions one of two ways:
1. Simple, textual completions are treated as editable tokens.
2. Completions with an element or more than one node are treated as readonly because they are arbitrary content we cannot be sure to support in the editor.

Does that make sense?

Reply:
Unsure. Perhaps this should be up to the author. E.g. inserting an image shouldn't be make non editable? Or is this for fragments?

Reply:
I am also not 100% sure. Currently, the idea was to make all non-text completions non-editable. My thinking comes from working with examples like the abbreviation completer shown in this PR description and thinking it is strange to allow editing because the user can edit text children but not attributes of the inserted `<abbr>` element. And with my limited experience with contenteditable, a user can get into strange places editing arbitrary HTML. So my first inclination was to pick a default that avoids that.

But you have more knowledge and wisdom about these things, and I'm interested in your take on my concerns.